packages/engine/Source/Core/EarthOrientationParameters.js

```diff
@@ -19,10 +20,6 @@ import TimeStandard from "./TimeStandard.js";
  * @constructor
```

Comment:
Should we add a warning here that this should typically not be constructed directly, like we do with `Model.js`?

https://github.com/CesiumGS/cesium/blob/b9106c0a915be1b59d1f35878ebbbe6f97505f43/packages/engine/Source/Scene/Model/Model.js#L42

Reply:
There are cases where the constructor may be used. I've add a disclaimer similar to `IonResource` - 

> This object is normally not instantiated directly, use {@link EarthOrientationParameters.fromUrl}.